microsoft-edge/webview2/how-to/self-hosting.md

```diff
@@ -0,0 +1,235 @@
+---
+title: Self-hosting by deploying preview channels
+description: How to set up self-hosting by deploying using the WebView2 preview channels, which are the Insider preview channels of Microsoft Edge (Canary, Dev, and Beta).
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.service: microsoft-edge
+ms.subservice: webview
+ms.date: 02/15/2024
+---
+# Self-hosting by deploying preview channels
+
+_Self-hosting_ means deploying a preview version of the WebView2 Runtime with your app.  This allows you and your development team to use your app daily with the preview version of the WebView2 Runtime, to find issues that will affect your particular WebView2 app, before the forthcoming changes in the WebView2 Runtime preview migrate to the Evergreen WebView2 Runtime.
+
+The self-hosting approach provides real-world usage of your app with the latest prerelease version of the WebView2 Runtime, to catch any issues that may arise.  Do this self-hosting approach in tandem with prerelease testing of your app, which instead provides a structured approach to testing your app with the latest version of the WebView2 Runtime.
+
+| Approach | Technique | Purpose/value |
+|---|---|---|
+| Prerelease testing | todo | todo |
```

Comment:
global in PR's changed files: find "todo".  @victorhuangwq 
Here, describe prerelease testing, so as to contrast w/ self-hosting, side-by-side in table

Reply:
closing.  found only 1 hit on todo in pr, asking if should remove tab-set w/ api links for the api approach.  d/k why we'd remove that (don't remember).  Seems clear we should keep tab-set.  removed "todo" comment there.  @victorhuangwq